app/Http/Controllers/JobController.php

```diff
@@ -165,11 +165,19 @@ function ($value, $key) {
                 'text' => $jobLang['apply']['edit_link_label'],
             ];
         } elseif (WhichPortal::isHrPortal()) {
-            $applyButton = [
-                'href' => route('hr_advisor.jobs.summary', $jobPoster->id),
-                'title' => null,
-                'text' => Lang::get('hr_advisor/job_summary.summary_title'),
-            ];
+            if ($jobPoster->hr_advisors->get($user->id)) {
```

Comment:
I'm not sure [get](https://laravel.com/docs/5.8/collections#method-get) is the correct method here. This will only work properly if the collection is keyed by object id, which I'm not sure is the case. I'd also recommend a `!== null` for explicitness. 

But instead, I'd use [contains](https://laravel.com/docs/5.8/collections#method-contains) with a key value pair, ie `$jobPoster->hr_advisors->contains('id', $user->id)`